### PR TITLE
Additional internal components

### PR DESCRIPTION
**Pull request by alse**
Additional internal components from #3:

Tables, Callouts, Lower page nav, Breadcrumbs, Internal version select, Internal sidebar menu

### CODE REVIEW

File: libs/design-system/src/lib/Components/InternalVersionSelect/InternalVersionSelect.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  versions: VERSIONS,
+  selectedVersionName: VERSIONS[0].name,
+};
+
+export const dark = Template.bind({});
+dark.parameters = {
```
Comment (aureentuluvacal) [inline libs/design-system/src/lib/Components/InternalVersionSelect/InternalVersionSelect.stories.tsx:29]:
```suggestion
dark.args = {
  versions: VERSIONS,
  selectedVersionName: VERSIONS[0].name,
};
dark.parameters = {
```

**Review by aureentuluvacal — CHANGES_REQUESTED** [review]
Works fantastically. Just have one callout. I'll make sure my storybook stories are just as clean lol

**Comment by alse** [comment]
@aureentuluvacal thanks, updated!

**Comment by alse** [comment]
Contents were moved to https://github.com/onflow/next-docs-v2/pull/2, closing

**Pull request closed without merging**